echopype/echodata/echodata.py

```diff
@@ -256,6 +256,42 @@ def __setattr__(self, __name: str, __value: Any) -> None:
                         self._tree[group_path].ds = __value
         super().__setattr__(__name, attr_value)
 
+    def _harmonize_env_param_time(
+        self, p: Union[int, float, xr.DataArray], ping_time: Union[xr.DataArray, datetime.datetime]
+    ):
+        """
+        Harmonize time coordinate between Beam_groupX data and env_params to make sure
+        the timestamps are broacast correctly in calibration and range calculations.
+
+        If timestamp exist for env_params (i.e. it is of type xr.DataArray),
+        they cpme from EchoData["Environment"] with coordinate time1.
+        For EK60, Environment.time1 = Beam_group1.ping_time, both from the RAW0 datagram
+        so are directly interchangeable.
+        For EK80, Environment.time1 is from the Environment XML datagram,
```

Comment:
I assume that's true also for AZFP? Plus, that's how you've implemented it, below. 

How about ES70, ES80 and EA640?

Reply:
ES70 in theory has the same format as EK60 but I have not actually handle an example file. ES80 and EA640 has the same format as EK80. I think we have ES80 and EA640 test files, and can try to run them through the calibration code to see what happens.

Reply:
I've updated the function so that `sonar_model` is no longer used.